sdk/netapp/Azure.ResourceManager.NetApp/src/Generated/Extensions/NetAppExtensions.cs

```diff
@@ -130,6 +130,32 @@ public static Response<NetAppCheckAvailabilityResult> CheckNetAppQuotaAvailabili
             return GetExtensionClient(subscriptionResource).CheckNetAppQuotaAvailability(location, content, cancellationToken);
         }
 
+        /// <summary>
+        /// Provides storage to network proximity and logical zone mapping information.
+        /// Request Path: /subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/regionInfo
+        /// Operation Id: NetAppResource_QueryRegionInfo
+        /// </summary>
+        /// <param name="subscriptionResource"> The <see cref="SubscriptionResource" /> instance the method will execute against. </param>
+        /// <param name="location"> The location. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public static async Task<Response<RegionInfo>> QueryRegionInfoNetAppResourceAsync(this SubscriptionResource subscriptionResource, AzureLocation location, CancellationToken cancellationToken = default)
```

Comment:
The name is weird, would like to have better name by using `override-operation-name`

Reply:
renamed